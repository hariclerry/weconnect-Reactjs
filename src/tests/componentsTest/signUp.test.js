import React from "react";
import {shallow, mount} from "enzyme";
import { MemoryRouter } from "react-router-dom";
import weconnectStore from "store";
import UserSignup from "containers/user/signup";


describe('UserSignup component',() =>{
  beforeEach(() => {
});
it('should match snapshot',() =>{
  const wrapper = shallow(<UserSignup/>)
  expect(wrapper).toMatchSnapshot();
})

it("should render the RegisterBusiness component", () => {
  const mountComponent = mount(<MemoryRouter><UserSignup store = {weconnectStore}/></MemoryRouter>)
  expect(mountComponent.length).toBe(1);
  
});

}

)