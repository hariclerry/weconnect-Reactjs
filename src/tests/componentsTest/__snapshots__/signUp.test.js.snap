// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserSignup component should match snapshot 1`] = `<Route />`;
